src/Card/Card.js

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { cardCtaOptions, cardAlignOptions } from "./Card.levers";
+import { FlexBox } from "../FlexBox/FlexBox";
+import { Heading } from "../Heading/Heading";
+import { CtaLink } from "../CtaLink/CtaLink";
+import { iconOptions } from "../common/icon/icon.levers";
+import { CtaButton } from "../CtaButton/CtaButton";
+import { ctaButtonVariants } from "../CtaButton/CtaButton.levers";
+import { buttonSizes } from "../common/button/button.levers";
+
+/**
+ * Card Component
+ *
+ * HTML card element
+ */
+export const Card = ({
+  className,
+  headline,
+  headingLevel,
+  superHeadline,
+  image,
+  text,
+  cta,
+  ctaText,
+  linkIcon,
+  buttonVariant,
+  buttonSize,
+  align,
+  isMinimal,
+  ...props
+}) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+  const levers = {};
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  let wrapperClasses =
+    "su-bg-white su-text-black su-border su-border-solid su-border-transparent-black su-shadow";
+
+  let buttonClasses = "su-rs-mt-neg1";
+
+  // Basic card image has aspect ratio 4x3 for non-round option
+  let imageWrapperClasses = "su-aspect-w-4 su-aspect-h-3";
```

Comment:
Please check with @kerri-augenstein to see what she'd like the default aspect ratio to be, and whether we want options for different aspect ratios 😺